packages/react-native-reanimated/apple/reanimated/Fabric/REAInitializerRCTFabricSurface.h

```diff
@@ -1,7 +1,7 @@
 #ifdef RCT_NEW_ARCH_ENABLED
 
-#import <RNReanimated/REAModule.h>
 #import <React/RCTFabricSurface.h>
+#import <reanimated/apple/REAModule.h>
```

Comment:
I've decided to add `apple/` in the path to avoid conflicts with `Common/cpp/reanimated`. Also, it looks cool.